fixtures/fun-generation/picospot-20-led.json

```diff
@@ -229,25 +236,85 @@
     "Gobo Wheel": {
       "capabilities": [
         {
-          "dmxRange": [0, 124],
+          "dmxRange": [0, 15],
           "type": "WheelSlot",
-          "slotNumberStart": 1,
-          "slotNumberEnd": 8,
-          "helpWanted": "At which DMX value can which gobo be selected? Are split positions between two gobos possible?"
+          "slotNumber": 1
+        },
+        {
+          "dmxRange": [16, 31],
+          "type": "WheelSlot",
+          "slotNumber": 2
         },
         {
-          "dmxRange": [125, 249],
+          "dmxRange": [32, 46],
+          "type": "WheelSlot",
+          "slotNumber": 3
+        },
+        {
+          "dmxRange": [47, 62],
+          "type": "WheelSlot",
+          "slotNumber": 4
+        },
+        {
+          "dmxRange": [63, 78],
+          "type": "WheelSlot",
+          "slotNumber": 5
+        },
+        {
+          "dmxRange": [79, 93],
+          "type": "WheelSlot",
+          "slotNumber": 6
+        },
+        {
+          "dmxRange": [94, 109],
+          "type": "WheelSlot",
+          "slotNumber": 7
+        },
+        {
+          "dmxRange": [110, 124],
+          "type": "WheelSlot",
+          "slotNumber": 8
+        },
+        {
+          "dmxRange": [125, 140],
           "type": "WheelShake",
-          "slotNumberStart": 2,
-          "slotNumberEnd": 8,
-          "helpWanted": "At which DMX value can which gobo be selected? Are split positions between two gobos possible?"
+          "slotNumber": 2
+        },
+        {
+          "dmxRange": [141, 156],
+          "type": "WheelShake",
+          "slotNumber": 3
```

Comment:
Shake speed slow to fast can be added according to https://github.com/OpenLightingProject/open-fixture-library/issues/3911 to all the shake capabilities.